src/main/resources/db/changelog/changes/2-ays-dml.xml

```diff
@@ -106,12 +106,30 @@
             <column name="LAST_NAME" value="Sistemi"/>
             <column name="COUNTRY_CODE" value="90"/>
             <column name="LINE_NUMBER" value="1234567890"/>
+            <column name="ROLE" value="ADMIN"/>
+            <column name="STATUS" value="ACTIVE"/>
+            <column name="CREATED_USER" value="AYS"/>
+            <column name="CREATED_AT" valueDate="now()"/>
+        </insert>
+
+        <insert tableName="AYS_ADMIN_USER">
+            <column name="ID" value="9262f0fc-93db-4f7e-81c6-aaad85c2b299"/>
```

Comment:
The approach of only changing the last 2 digits of the ID is not a correct one. Can we write the following ID value instead?

`ea29a0bd-ad5d-4068-88c3-a8835b8d555e`

Reply:
I just revised the `ID` field. Can we look through it again?